src/components/menu/Menu.tsx

```diff
@@ -33,20 +32,20 @@ const Menu = () => {
           ))}
         </div>
       </div>
-      <Button btnType="secondary-outlined" className="w-32" onClick={() => setIsOpen(true)}>
-        <MoreHorizSvg className="h-6 w-6"/>
+      {/* <Button btnType="secondary-outlined" className="w-32" onClick={() => setIsOpen(true)}>
+        <MoreHorizSvg className="h-6 w-6" />
         Show Menu
       </Button>
-      <Drawer isOpen={isOpen} setIsOpen={setIsOpen}/>
-      {/* <Collapsible
+      <Drawer isOpen={isOpen} setIsOpen={setIsOpen} /> */}
+      <Collapsible
```

Comment:
I dont know why you are using `Collapsible` here, use `Animate` directly because a drawer is not a collapsible